projects/netgrif-components-core/src/lib/authentication/profile/models/user-change-password-request.ts

```diff
@@ -0,0 +1,5 @@
+export interface UserChangePasswordRequest {
+    login: string;
```

Comment:
why are we sending the login? Are we not always changing the password of the current user?